Sodium/PWHash.swift

```diff
@@ -15,18 +15,18 @@ public struct PWHash {
 
 extension PWHash {
     public enum Alg {
-        case Default
-        case Argon2I13
-        case Argon2ID13
+        case `default`
+        case argon2I13
+        case argon2ID13
     }
 }
 
 extension PWHash.Alg {
     var id: Int32 {
         switch self {
-        case .Default:    return crypto_pwhash_alg_default()
-        case .Argon2I13:  return crypto_pwhash_alg_argon2i13()
-        case .Argon2ID13: return crypto_pwhash_alg_argon2id13()
+        case .`default`:    return crypto_pwhash_alg_default()
```

Comment:
Avoid ticks and name it `defaultAlgorithm` as @jedisct1 [suggested](https://github.com/jedisct1/swift-sodium/pull/184#issuecomment-489597858).